contributors/design-proposals/node/accelerator-monitoring.md

```diff
@@ -75,7 +75,7 @@ From the summary API, they will flow to heapster and stackdriver.
 
 ## Caveats
 - As mentioned before, this would add a requirement that cAdvisor and kubelet are dynamically linked.
-- We would need to make sure that kubelet is able to access the nvml libraries. Some existing container based nvidia driver installers install drivers in a special directory. We would need to make sure that that directory is in kubelet’s `LD_LIBRARY_PATH`.
+- We would need to make sure that kubelet is able to access the nvml libraries. Some existing container based nvidia driver installers install drivers in a special directory. We would need to make sure that directory is in kubelet’s `LD_LIBRARY_PATH`.
```

Comment:
@xichengliudui Thanks for the PR!

I've noticed that you've sent PRs to remove similar duplicate words before (https://github.com/kubernetes/community/pull/2826). Are you using a script or an automated way to find the duplicate words?

If so, it would be really helpful if you could fix all of the "duplicate word" typos in a single PR! :)

Reply:
@nikhita   Ok, thank you, but I didn't use a script, I found it by accident